web/src/pages/HomePage/HomePage.tsx

```diff
@@ -108,47 +110,90 @@ const HomePage = () => {
           <h3>How it Works</h3>
           <ol>
             <li>
+              <h4>Register</h4>
               <a
                 href={Constants.APPLICATION_URL}
                 target="_blank"
                 rel="noreferrer"
               >
-                Register for the competition by 11:59pm PT, August 16, 2023.
+                Register to participate.
               </a>{' '}
-              This includes a short video (roughly 3 minutes or less),
-              introducing yourself, explaining your ideas, how you will use
-              RedwoodJS, and anything else you might want us to know.
+              You'll receive:
+              <ul>
+                <li>An invitation to a weekly virtual meetup (optional)</li>
+                <li>Access to a private Discord channel</li>
+                <li>Connect with other participants and build a team</li>
+                <li>FREE Sticker</li>
+                <li>
+                  Entry to our raffle and win 2{' '}
+                  <a
+                    href={Constants.CONFERENCE_URL}
+                    target="_blank"
+                    rel="noreferrer"
+                  >
+                    RedwoodJS Conference Tickets
+                  </a>
+                </li>
+              </ul>
```

Comment:
```suggestion
              </ul>
              Register now using
                            <a
                 href={Constants.APPLICATION_URL}
                 target="_blank"
                 rel="noreferrer"
               >
               this form</a>.
```

You'll need to manage styles/font here I think.